features/providers/review_and_print.feature

```diff
@@ -128,13 +128,75 @@ Feature: Review and print your application
-     | Disregarded benefits total |
-     | Financial help from friends or family total |
-     | Maintenance payments from a former partner total |
-     | Income from a property or lodger total |
-     | Pension total |
+      | question |
+      | Benefits total |
+      | Disregarded benefits total |
+      | Financial help from friends or family total |
+      | Maintenance payments from a former partner total |
+      | Income from a property or lodger total |
+      | Pension total |
+      | Student loan |
+
+    And I should not see "Housing benefit total"
+
+    And I should see "Housing payments"
+    And I should see "Childcare payments"
+    And I should see "Maintenance payments to a former partner"
+    And I should see "Payments towards legal aid in a criminal case"
+
+  Scenario: For a non-passported truelayer bank transactions journey
+    Given I have completed truelayer application with merits and no student finance
+    When I view the review and print your application page
+
+    Then the following sections should exist:
+      | tag | section |
+      | h2  | Client details |
+      | h2  | What you're applying for |
+      | h2  | Delegated functions |
+      | h2  | Covered under an emergency certificate |
+      | h2  | Covered under a substantive certificate |
+      | h2  | Emergency cost limit |
+      | h2  | Income, regular payments and assets |
+      | h3  | Income |
+      | h3  | Regular payments |
+      | h3  | Property |
+      | h3  | Vehicles |
+      | h2  | Which bank accounts does your client have? |
+      | h3  | Does your client have any savings accounts they cannot access online? |
+      | h2  | Which savings or investments does your client have? |
+      | h2  | Which assets does your client have? |
+      | h2  | Restrictions on your client's assets |
+      | h2  | Payments from scheme or charities |
+      | h2  | Case details |
+      | h2  | Latest incident details |
+      | h2  | Opponent details |
+      | h2  | Variation or discharge under section 5 protection from harassment act 1997r |
+      | h2  | Extend, variation or discharge - Part IV |
+      | h1  | Print your application |
+
+    Then the following sections should not exist:
+      | tag | section |
+      | h2  | Your client's income |
+      | h3  | Employment income |
+      | h3  | Payments your client receives |
+      | h3  | Payments your client receives in cash |
+      | h3  | Student finance |
+      | h2  | Your client's outgoings |
+      | h3  | Payments your client makes |
+      | h3  | Payments your client makes in cash|
+      | h2  | Your client's capital |
+
+    And the "Income, regular payments and assets" review section should contain:
+      | question |
+      | Benefits total |
+      | Disregarded benefits total |
+      | Financial help from friends or family total |
+      | Maintenance payments from a former partner total |
+      | Income from a property or lodger total |
+      | Pension total |
+
```

Comment:
This appears to be a repetition of above scenario, so wonder if it is worth it - might be clearer/simpler to just focus on the student loan aspect?!